tools/test-harnesses/src/types/form/formUtils.types.ts

```diff
@@ -0,0 +1,26 @@
+export interface FormUtils<T = string> {
+  /**
+   * Returns whether the input is disabled
+   */
+  isDisabled: () => boolean;
+
+  /**
+   * Returns whether the input state is `valid`
+   */
+  isValid: () => boolean;
+
+  /**
+   * Returns whether the input state is `error`
+   */
+  isError: () => boolean;
+
+  /**
+   * Returns the input value
+   */
+  getInputValue: () => T;
```

Comment:
Does `T = string` mean this would break for DatePicker, NumperInput etc?

Reply:
doesn't `T = string` mean that it is the default but can be overridden?

Reply:
Ah, yeah I've always used it with `extends` so forgot that this also works

Reply:
i should add  `extends`

Reply:
I think it's fine as is. You'd have to extend `any` for this I think anyway